src/App.js

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+  const [entries, setEntries] = useState(chatMessages);
+
+  const calcTotalLikes = () => {
```

Comment:
Oh I really like how you used the built-in `reduce` function coupled with the ternary, makes your code so concise!